buddyboss/buddyboss-platform
# PROD-718 #3066 - fix: Text format is being removed when pasting it in compose message or activity

### All Submissions:






### Changes proposed in this Pull Request:



Fixes #3066 .

### How to test the changes in this Pull Request:

1. Copy any random text with format e.g. with break line at least.
2. Paste it in compose a message/activity/discussion/reply.
3. You'll see that it will paste as formatted text.

### Proof Screenshots or Video
https://vimeo.com/643521506/236d436ad7


### Other information:




### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.
